server/xpub/entities/manuscript/resolvers.test.js

```diff
@@ -275,29 +283,29 @@ describe('Submission', () => {
 
   describe('uploadManuscript', () => {
     // TODO subscribe to uploadProgress before this or mock
-    it.skip('extracts title from PDF', async () => {
-      const { id } = await Manuscript.save(Manuscript.new())
-      const file = {
-        filename: 'manuscript.pdf',
-        stream: fs.createReadStream(
-          `${__dirname}/../../../../test/fixtures/dummy-manuscript-2.pdf`,
-        ),
-        encoding: 'utf8',
-        mimetype: 'application/pdf',
-        size: 73947,
-      }
-      const manuscript = await Mutation.uploadManuscript(
-        {},
-        { id, file, fileSize: file.size },
-      )
-      expect(manuscript).toMatchObject({
-        id,
-        meta: {
-          title:
-            'The Relationship Between Lamport Clocks and Interrupts Using Obi',
-        },
-        files: [{ filename: 'manuscript.pdf' }],
-      })
-    })
+    // it.skip('extracts title from PDF', async () => {
```

Comment:
Warnings don't fail the build, so you don't need to comment this out

Reply:
But it does stop you making a commit ;P

Reply:
Ah I didn't realise, I may not be running the commit hook locally... I suggest deleting the test and addressing it in #493

Reply:
Good plan!